mirdata/datasets/billboard.py

```diff
@@ -0,0 +1,480 @@
+from mirdata import annotations
+from mirdata import io
+
+BIBTEX = """
+@inproceedings{burgoyne_billboard,
+author = {Burgoyne, John Ashley and Wild, Jonathan and Fujinaga, Ichiro},
+year = {2011},
+title = {An {Expert} {Ground} {Truth} {Set} for {Audio} {Chord} {Recognition} and {Music} {Analysis}},
+booktitle={Proceedings of the 12th International Society for Music Information Retrieval Conference, ISMIR}
+}
+
+@phdthesis{phdthesis,
+  author       = {Burgoyne, John Ashley}, 
+  title        = {Stochastic {Processes} and {Database}-{Driven} {Musicology}},
+  school       = {McGill University, Montréal, Québec},
+  year         = 2012,
+}
+"""
+
+REMOTES = {
+    "metadata": download_utils.RemoteFileMetadata(
+        filename="billboard-2.0-index.csv",
+        url="https://www.dropbox.com/s/o0olz0uwl9z9stb/billboard-2.0-index.csv?dl=1",
+        checksum="c47d304c212725998839cf9bb1a417aa",
+    ),
+    "annotation_salami": download_utils.RemoteFileMetadata(
+        filename="billboard-2.0-salami_chords.tar.gz",
+        url="https://www.dropbox.com/s/2lvny9ves8kns4o/billboard-2.0-salami_chords.tar.gz?dl=1",
+        checksum="6954a6fad962a111e69c9c80cb87d3a5",
+    ),
+    "annotation_lab": download_utils.RemoteFileMetadata(
+        filename="billboard-2.0.1-lab.tar.gz",
+        url="https://www.dropbox.com/s/t390alzrkx0c9yt/billboard-2.0.1-lab.tar.gz?dl=1",
+        checksum="a7b1fa6a7e454bf73ced7c29207aa597",
+    ),
+    "annotation_mirex13": download_utils.RemoteFileMetadata(
+        filename="billboard-2.0.1-mirex.tar.gz",
+        url="https://www.dropbox.com/s/fg8lvy79o7etiyc/billboard-2.0.1-mirex.tar.gz?dl=1",
+        checksum="97e5754699f3b45aa5cc70d8a7611c54",
+    ),
+    "annotation_chordino": download_utils.RemoteFileMetadata(
+        filename="billboard-2.0-chordino.tar.gz",
+        url="https://www.dropbox.com/s/e9dm23vbawg9dsw/billboard-2.0-chordino.tar.gz?dl=1",
+        checksum="530218e8d7077bbd4b08b45f447f5e8f",
+    ),
+}
+
+LICENSE_INFO = "Creative Commons Attribution 4.0 International Public License."
+
+
+class Track(core.Track):
+    """McGill Billboard Dataset Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        track_id (str): track id
+        audio_path (str): audio path of the track
+        chords_full (ChordData): HTK-style LAB files for the chord annotations (full)
+        chords_majmin7 (ChordData): HTK-style LAB files for the chord annotations (majmin7)
+        chords_majmin7inv (ChordData): HTK-style LAB files for the chord annotations (majmin7inv)
+        chords_majmin (ChordData): HTK-style LAB files for the chord annotations (majmin)
+        chords_majmininv (ChordData): HTK-style LAB files for the chord annotations(majmininv)
+        chroma (np.array): Array containing the non-negative-least-squares chroma vectors
+        tuning (list): List containing the tuning estimates
+        sections (SectionData): Letter-annotated section data (A,B,A')
```

Comment:
This one also is a cached property

Reply:
Fixed in https://github.com/Spijkervet/mirdata/commit/9194bc15e9b2faa1210021185dec736db953184f

Reply:
> Just a nit comment on the table: if you're starting the name of the dataset with McGill maybe is worth moving it to match the alphabetical ordering... though it won't be the same order as the loaders (since the loader is called billboard), so what about calling it `Billboard (McGill)` or similar in the table?

Agreed, fixed in https://github.com/Spijkervet/mirdata/commit/12ba21287055322b4f81f96e8cbf8965383a5b80